Implement exclusion filter (#175)
Navideck/universal_ble: CHANGELOG.md

## 0.21.1
* Fix device name resolution on Windows

> * Add `exclusionFilters` for scan, allows excluding devices from scan results

Reviewer: ```suggestion
* Add `exclusionFilters` to filter out devices from scan results
```
